conf/server/server_full.conf

```diff
@@ -102,6 +102,19 @@ server {
     # Format of logs, <text|json>. Default: text.
     # log_format = "text"
 
+    # auth_policy_engine: The auth policy engine used for authorization decision
+    # For more details, refer to doc/authorization_policy_engine.md
+    # auth_policy_engine {
+    #     # Indicates that OPA provided through local files be used to 
+    #     # instantiate the auth policy engine
+    #     opa_file_provider {
```

Comment:
>The fact that we are using OPA internally feels like a little bit of an implementation detail... is it "just" rego evaluation, or are there OPA-specific things here? Would it make sense for the name of this provider to be something about rego instead?

I think rego is really only tied to only OPA right now, the evaluation semantics are still all OPA specific, i.e. handling of errors, execution order, etc. So I think in this case "opa" may be a more direct translation of what's happening since it encapsulates both the static and dynamic semantics. 

>It would be nice if the name was a bit shorter too ... the provider suffix is probably redundant, perhaps something like rego_file or opa_file
>I imagine that if we had rego_file, then we may in the future have something like local_opa etc?

Yea, I really like `local_opa` even now, since most likely the next extension to this is `remote_opa` where policy can be defined in a remote server. What do you think about this?